spec/steps/medicaid/contact_phone_spec.rb

```diff
@@ -2,13 +2,10 @@
 
 RSpec.describe Medicaid::ContactPhone do
   describe "Validations" do
-    it "validates presence" do
+    it "does not validates presence" do
```

Comment:
could we remove this test entirely? think this is useful to drive out the change, but like only keeping around positive assertions for posterity!

Reply:
yeah good call,I was 50/50 on this. removed at 8bac807a